app/views/events/index.html.erb

```diff
@@ -26,14 +26,19 @@
           <td>
             <% if event.has_member?(current_user) %>
               <%= t('.you_are_member') %>
-              <%= link_to t('links.delete_me'), event_members_path(event), class: 'btn btn-info', method: :delete %>
+              <%= link_to t('links.delete_me'), event_members_path(event), class: 'btn btn-info', method: :destroy %>
```

Comment:
There is not HTTP method `destroy` we talk about that some time ago. You need to have `delete` here.